api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/DatinCodeDataSet.java

```diff
@@ -22,8 +22,8 @@ public DataSetDefinition constructDataset(List<Parameter> list) {
         "select p.patient_id,la.value_reference DatimCode from patient p "
             + "inner join encounter e on p.patient_id=e.patient_id "
             + "inner join location l on l.location_id=e.location_id "
-            + "inner join location_attribute la on la.location_id and l.location_id "
-            + "where la.attribute_type_id =2 and la.location_id=:location and la.voided=0 "
+            + "inner join location_attribute la on la.location_id and l.location_id = :location "
```

Comment:
@nelsonhomo  solicito rever esse trecho da query, temos :
inner join location_attribute la on **la.location_id and** l.location_id = :location

Reply:
@Guimino corrigido